controllers/space/SpaceController.php

```diff
@@ -2,44 +2,67 @@
 
 namespace humhub\modules\smartVillage\controllers\space;
 
+use Firebase\JWT\JWT;
+use humhub\modules\rest\models\ConfigureForm;
 use humhub\modules\smartVillage\components\AuthBaseController;
 use humhub\modules\space\models\Space;
 use humhub\modules\rest\definitions\SpaceDefinitions;
+use humhub\modules\user\models\User;
 
 class SpaceController extends AuthBaseController
 {
     public function actionFind(){
         $results = [];
-        $query = Space::find()->where(['visibility' => Space::VISIBILITY_ALL]);
+
+        //Check the requested user is guest or not
+        $user = $this->checkUserIsRegistered();
```

Comment:
something seems to be still bad with the space controller:

```
{
	"name": "Exception",
	"message": "Class 'humhub\\modules\\smartVillage\\controllers\\space\\Yii' not found",
	"code": 0,
	"type": "Error",
	"file": "/var/www/localhost/htdocs/protected/humhub/modules/smart-village-app-humhub-ext/controllers/space/SpaceController.php",
	"line": 55,
	"stack-trace": [
		"#0 /var/www/localhost/htdocs/protected/humhub/modules/smart-village-app-humhub-ext/controllers/space/SpaceController.php(18): humhub\\modules\\smartVillage\\controllers\\space\\SpaceController->checkUserIsRegistered()",
		"#1 [internal function]: humhub\\modules\\smartVillage\\controllers\\space\\SpaceController->actionFind()",
		"#2 /var/www/localhost/htdocs/protected/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)",
		"#3 /var/www/localhost/htdocs/protected/vendor/yiisoft/yii2/base/Controller.php(181): yii\\base\\InlineAction->runWithParams(Array)",
		"#4 /var/www/localhost/htdocs/protected/vendor/yiisoft/yii2/base/Module.php(534): yii\\base\\Controller->runAction('find', Array)",
		"#5 /var/www/localhost/htdocs/protected/vendor/yiisoft/yii2/web/Application.php(104): yii\\base\\Module->runAction('smartVillage/sp...', Array)",
		"#6 /var/www/localhost/htdocs/protected/vendor/yiisoft/yii2/base/Application.php(392): yii\\web\\Application->handleRequest(Object(humhub\\components\\Request))",
		"#7 /var/www/localhost/htdocs/index.php(25): yii\\base\\Application->run()",
		"#8 {main}"
	]
}
```

calendar and post are working.
any idea @gaurav-handysolver?

Reply:
`use Yii` was missing